### ISSUE TITLE
Investigate upgrade VS Code extension (0.2.0) to use latest source code version 0.9.3

### ISSUE THREAD

**Issue by mkcomer** [inclusive-dev-tools/woke#13]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by mkcomer** [comment]
There is not going to be a PR for this issue.

After investigating and building the VS-code/woke locally and debugging the extension, the package.json which controls all dependencies does not need to be updated to the latest version of woke.

The vs-code woke extension doesn't manage the installation of woke, you need to install it yourself and it will use whatever it can find in your $PATH, or what you define in the extension setting woke.executablePath.

**Comment by mkcomer** [comment]
*Based on how provider.ts parses the response from the woke binary, I don’t believe any feature enhancements to the woke source code will not be compatible with the VS code extension.

VS-code-woke and woke are not version controlled with the same release cycle – a bit confusing. VS-Code-Woke @ 0.2.0 and 
Woke @0.10.0.

**Comment by mkcomer** [comment]
The woke extension is supposed to prompt you to upgrade your woke binary if it is out of date. I found, however, running the vs-code-woke on woke@0.9.0 did not prompt me to upgrade my woke binary (woke is @ 0.10.0).

**Comment by mkcomer** [comment]
In light of investigation, suggested added stories to backlog for development on vs-code-woke:

1. Fix disableVersionCheck – it is supposed to warn you if your woke version is outdated when running your vs-code extension. I did not find this to work. I ran VS Code extension on woke 0.9.0 and woke is on version 0.10.0

2. Add debugging to Vs-code/woke to log out version of woke you are running. 

3. Enable the same version control between all woke repositories to avoid confusion – automatically update vs-code-woke version whenever woke is upgraded to new versions
